src/machines/worker.test.ts

```
@@ -0,0 +1,104 @@
+
+const SOCKET_URL = 'wss://sync.example.com/ws'
+const socketEndpoint = ws.link('wss://sync.example.com/ws')
+export const server = setupServer(
+	socketEndpoint.addEventListener('connection', (server) => {
+		server.server.connect()
+	}),
+	// We shouldn't need this but msw complains (without an error) if we don't have it
+	http.get(SOCKET_URL.replace('wss', 'https'), () => {
+		return new Response()
+	})
+)
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)